crates/router/src/connector/adyen/transformers.rs

```diff
@@ -2040,6 +2052,188 @@ impl From<AdyenNotificationRequestItemWH> for AdyenResponse {
             refusal_reason: None,
             refusal_reason_code: None,
             additional_data: None,
+            auth_code: None,
+        }
+    }
+}
+
+// Payouts
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AdyenPayoutResponse {
+    psp_reference: String,
+    result_code: AdyenStatus,
+    amount: Option<Amount>,
+    merchant_reference: Option<String>,
+    refusal_reason: Option<String>,
+    refusal_reason_code: Option<String>,
+    additional_data: Option<AdditionalData>,
+    auth_code: Option<String>,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AdyenPayoutEligibilityRequest {
+    pub amount: Amount,
+    pub merchant_account: String,
+    pub payment_method: PayoutCardDetails,
+    pub reference: String,
+    pub shopper_reference: String,
+}
+
+#[derive(Default, Debug, Serialize, Deserialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct PayoutCardDetails {
+    #[serde(rename = "type")]
+    pub _type: String,
+    pub number: String,
+    pub expiry_month: String,
+    pub expiry_year: String,
+    pub holder_name: String,
```

Comment:
```suggestion
    pub number: Secret<String>,
    pub expiry_month: Secret<String>,
    pub expiry_year: Secret<String>,
    pub holder_name: Secret<String>,
```

Reply:
This will be done in a separate PR, along with vault related changes for masking sensitive data